### PR TITLE
Sanity metadata

### PR DESCRIPTION
**Pull request by Aerilym**

### CODE REVIEW

File: packages/sanity-cms/lib/metadata.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          }
+        : {
+            type: 'website',
+          }),
+    },
+  };
+}
+
+function generateMetaImage(client: SessionSanityClient, image: NonNullable<SeoType['metaImage']>) {
+  return getSanityImageUrlBuilder(client, image).width(1200).height(630).fit('crop').url();
```
Comment (yougotwill) [inline packages/sanity-cms/lib/metadata.ts:80]: Should these be constants?

File: packages/sanity-cms/components/SanityPdf.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  src: string;
+  url: URL;
+  strings: {
+    openPdfInNewTab: string;
+    openPdfInNewTabAria: string;
+  };
+};
+
+export default function SanityPdf({ src, url, strings }: SanityPdfProps) {
+  if (!url || !src) return null;
```
Comment (yougotwill) [inline packages/sanity-cms/components/SanityPdf.tsx:14]: Should we not do a file not found or log something here?

File: packages/sanity-cms/components/SanityFileDownload.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    clickToDownloadAria: string;
+    openPdfInNewTab: string;
+    openPdfInNewTabAria: string;
+  };
+};
+
+export default function FileDownload({ fileName, src, strings }: FileDownloadProps) {
+  const [downloaded, setDownloaded] = useState(false);
+  const router = useRouter();
+  if (!src || !fileName) return null;
```
Comment (yougotwill) [inline packages/sanity-cms/components/SanityFileDownload.tsx:23]: Again should we not have some sort of file not found or error logging here?

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

File: packages/sanity-cms/components/SanityPdf.tsx (hunk 2 above)
Comment (Aerilym) [reply packages/sanity-cms/components/SanityPdf.tsx:14]: Nah not here, its on the client-side, we have logging on the page generation and notFound if its not there

File: packages/sanity-cms/components/SanityFileDownload.tsx (hunk 3 above)
Comment (Aerilym) [reply packages/sanity-cms/components/SanityFileDownload.tsx:23]: Nah not here, its on the client-side, we have logging on the page generation and notFound if its not there

File: packages/sanity-cms/lib/metadata.ts (hunk 1 above)
Comment (Aerilym) [reply packages/sanity-cms/lib/metadata.ts:80]: Only used once here, IMO not worth abstracting

**Pull request merged**